src/Bundle/Maker/MakeFactory.php

```diff
@@ -80,6 +75,10 @@ public function interact(InputInterface $input, ConsoleStyle $io, Command $comma
             $io->newLine();
         }
 
+        if ($input->getArgument('class')) {
```

Comment:
I think previous notes should be displayed even if a class is passed as an argument to the command